src/components/Navigation.vue

```diff
@@ -0,0 +1,167 @@
+<template>
+  <div>
+    <div class="nav-bar">
+      <div id="logo">
+        <img src="@/assets/Vector.png" width="154.82" height="18.88" />
+      </div>
+      <div class="left-side">
+        <ul>
+          <li>
+            <a href="/">Movies  v</a>
```

Comment:
Export icon like this and use it instead of character "v".
![Uploading Screenshot 2021-11-09 at 19.31.52.png…]()

Reply:
Remove href='/' for now component should not redirect anywhere.